### ISSUE TITLE
[iOS] Exception on Time Picker when using SelectedTime

### ISSUE THREAD

**Issue by IlSocio** [aritchie/userdialogs#169]
1) Please check all of the platforms you are having the issue on
 [x] iOS
 [ ] Android
 [ ] UWP
 [ ] PCL

2) Please state the issue to the best of your ability below
The TimePromptConfig.SelectedTime raises an exception:
```
 TimePromptConfig config = new TimePromptConfig();
 config.SelectedTime = TimeSpan.FromHours(2);
 TimePromptResult res = await UserDialogs.Instance.TimePromptAsync(config);
```

3) Please copy and paste the exception details below.  If you are having a crash and this detail is missing, your issue will be closed without warning.
The exception below is thrown:
` "DateTimeKind.Unspecified cannot be safely converted"`

It seems to be related to the conversion between DateTime and NSDate
https://github.com/aritchie/userdialogs/blob/master/src/Acr.UserDialogs.iOS/TimePickerController.cs#L33
which needs to be handled properly:
http://stackoverflow.com/questions/27925179/converting-nullable-datetime-to-nsdate-after-switching-to-unfied-api

**Comment by aritchie** [comment]
I have a fix for this that I'll push out today.

**Comment by aritchie** [comment]
released with 5.2
